viral/wheel_freeze_movement.py

```
@@ -0,0 +1,187 @@
+
+def compare_to_suite2p_motion(motion_energy_pre: np.ndarray) -> None:
+    s2p_path = Path("/Volumes/MarcBusche/Josef/2P/2026-05-13/J030/suite2p/plane0")
+    stat = np.load(s2p_path / "stat.npy", allow_pickle=True)
+    ops = np.load(s2p_path / "ops.npy", allow_pickle=True).item()
+    motion_artifact = np.abs(np.diff(ops["xoff"]))
+    motion_artifact = ops["corrXY"]
+
+    motion_artifact_smoothed = moving_average(motion_artifact, window=10)
+    motion_energy_pre = moving_average(motion_energy_pre, window=10)
+
+    fig, ax1 = plt.subplots()
```

Ruff (0.16.0)

[warning] Unpacked variable `fig` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)